src/Toml/Bi/Monad.hs

```diff
@@ -22,7 +22,7 @@ import Data.Coerce (Coercible, coerce)
 
 In practice instead of @r@ we will use some @Reader Toml@ and instead of @w@ we will
 use @State Toml@. This approach with the bunch of utility functions allows to
-have single description for from/to 'Toml' conversion.
+have single description for from/to @TOML@ conversion.
```

Comment:
Here, it is emphasized, but above is not.

Reply:
I used `@` in some places but then got tired... This doesn't really matter much, just some formatting :slightly_smiling_face: